hdc/algo/ops/stats.py

```diff
@@ -187,16 +190,17 @@ def gammastd(x, nodata, cal_start, cal_stop, a=0, b=0):
     if alpha == 0 or beta == 0:
         return np.full_like(x, nodata, dtype="float64")
 
-    y = np.full(t, p_zero, dtype="float64")  # type: ignore
+    y = np.full(t, nodata, dtype="float64")  # type: ignore
 
     for ix in range(t):
-        if x[ix] > 0:
+        if x[ix] >= 0:
             y[ix] = p_zero + (
                 (1 - p_zero)
                 * sc.gammainc(alpha, x[ix] / beta)  # pylint: disable=no-member
```

Comment:
When `x[ix] == 0`, it computes `sc.gammainc(alpha, 0)` which always returns `0`. Thus, we still have the former intended behaviour `y[ix] = p_zero` if `x[ix] == 0` and it also handles `nodata < 0` in the meantime.

Reply:
ok what if `nodata > 0`?